src/surfing_penguin/db_interface.py

```diff
@@ -44,24 +44,31 @@ def load_user(id):
     return session.query(User).get(int(id))
 
 
-class Qstnr(object):
-    def __init__(self):
+class SurveyFunctions(object):
+    def __init__(self, name):
         self.iter = 0
-        self.questions = []
+        self.name = name
+        survey = Survey(name)
```

Comment:
Why create a Survey here? It seems to me that SurveyFunctions is not for creating a survey but for doing survey related functions.

Reply:
If db_interface would be more complicated in the future, you could break it into several files under db_interface folder. SurveyFunctions doesn't have to be a class; it could be a file with different operations to the database.

Reply:
@zfrank7777  Are you going to do this in this PR?